product_docs/docs/pgd/5/cli/installing/tpa.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Installing PGD CLI with TPA"
+navTitle: "TPA"
+description: "Installing PGD CLI with Trusted Postgres Architect"
+---
+
+By default, Trusted Postgres Architect installs and configures PGD CLI on each PGD node. 
+
+If you want to install PGD CLI on any non-PGD instance in the cluster, attach the pgdcli role to that instance in Trusted Postgres Architect's configuration file before deploying. 
+
+See [Trusted Postgres Architect](/tpa/latest/) for more information.
+
+[Next: Using PGD CLI](../using_cli)
```

Comment:
The embeded "next" tile already points to that topic, so that seems superfluous here?

Reply:
```suggestion
```